src/chi-vue/src/components/data-table-filters/AdvancedFilters.tsx

```diff
@@ -86,129 +91,119 @@ export default class AdvancedFilters extends Vue {
     );
   }
 
-  _createInputFilter(filter: DataTableFilter, index: number) {
+  async _changeFormElementFilter(ev: Event, elementType: DataTableFormElementFilters) {
+    if (ev.target) {
+      const newFilterData: {
+        name: string;
+        checked?: boolean;
+        value?: string;
+        id: string;
+      } | null = getElementFilterData(ev, elementType);
+
+      await this.storeModule.updateFilterConfig({
+        ...this.filterElementValue,
+        ...{
+          [newFilterData?.id.replace(/-desktop|-mobile/gi, '') || 'no-id']:
+            elementType !== 'checkbox' ? newFilterData?.value : newFilterData?.checked,
+        },
+      });
+
+      if (newFilterData && this.advancedFiltersData) {
+        updateFilterData(this.advancedFiltersData, newFilterData);
+      }
+    }
+  }
+
+  _createInputFilter(filter: DataTableFilter) {
     return (
       <div
         id={this._calculateFilterId(filter.name)}
         // Todo
         class={`
-          ${FORM_CLASSES.FORM_ITEM}
-          ${index !== 0 && !filter.advanced ? 'chi-data-table-filter__input -ml--2' : ''}`}>
+              ${FORM_CLASSES.FORM_ITEM}`}>
+        {this.mobile && (
+          <label for={filter.id} class={FORM_CLASSES.LABEL}>
```

Comment:
This looks good as it renders the form items with in Drawer with respective labels, but looks like we lack the same for rendering the standard filters in the Drawer. Here is a screenshot:
<img width="549" alt="Screenshot 2021-07-06 at 20 28 31" src="https://user-images.githubusercontent.com/25136131/124649531-ea2ac980-de98-11eb-836d-d0e77723b0b4.png">